shiny/api-examples/nav_insert/app-express.py

```diff
@@ -0,0 +1,53 @@
+from shiny import reactive
+from shiny.express import input, ui
+
+with ui.layout_sidebar():
+    with ui.sidebar():
+        ui.input_action_button("add", "Add 'Dynamic' tab")
+        ui.input_action_button("removeFoo", "Remove 'Foo' tabs")
+        ui.input_action_button("addFoo", "Add New 'Foo' tab")
+
+    with ui.navset_tab(id="set"):
+        with ui.nav_panel("Hello", value="Hello"):
+            "This is the hello tab"
+        with ui.nav_panel("Foo", value="Foo"):
+            "This is the Foo tab"
+        with ui.nav_menu("Static", value="tabs"):
+            with ui.nav_panel("Static 1", value="s1"):
+                "Static 1"
+            with ui.nav_panel("Static 2", value="s2"):
+                "Static 2"
```

Comment:
Let's make sure things like `id` and `value`s are consistent between the Core and Express versions